docs/configuration_requirements.rst

```diff
@@ -0,0 +1,19 @@
+Configuration requirements
+**************************
+
+.. only:: iso
+
+    This page shows the configuration requirements for the Inmanta Service Orchestrator |version_major| version.
+    These constraints are defined for the whole lifetime of this major version i.e.
+    they are not pinned to a specific three-digit version.
+
+
+    .. datatemplate:json:: required_configuration.json
+       :template: iso_config_requirements.tmpl
+
+.. only:: oss
+
+    This page shows the configuration requirements for version |version_major| of Inmanta.
+
+    .. datatemplate:json:: required_configuration.json
+       :template: oss_config_requirements.tmpl
```

Comment:
Why not split it in two templates? One for the "system requirements" (Python, RHEL, ...) and one for the components? Then we can put the latter in a `.. only:: iso` block but the former would be identical.